### ISSUE TITLE
[sass|scss] Shadow-piercing descendant combinator not supported

### ISSUE THREAD

**Issue by mattmischuk** [sasstools/sass-lint#974]
A fatal error is returned if `>>>` selector is used (shadow-piercing descendant combinator).

https://drafts.csswg.org/css-scoping/#deep-combinator

What version of Sass Lint are you using?
v1.10.2

Please include any relevant parts of your configuration

What did you do? Please include the actual source code causing the issue.

```scss
:host >>> .data-table { width: 100%; }
```

What did you expect to happen?

The Sass Lint would accept this type of combinator.

What actually happened? Please include any error messages given to you by Sass Lint.

```bash
$ sass-lint ./src/components/example/example.component.scss
```

```bash
/Users/example/node_modules/sass-lint/index.js:313
    throw new exceptions.SassLintFailureError(errorCount.count + ' errors were detected in \n- ' + errorCount.files.join('\n- '));
    ^
SassLintFailureError: 1 errors were detected in 
- ./src/components/example/example.component.scss
    at Function.sassLint.failOnError (/Users/example/node_modules/sass-lint/index.js:313:11)
    at detectPattern (/Users/example/node_modules/sass-lint/bin/sass-lint.js:31:10)
    at /Users/example/node_modules/sass-lint/bin/sass-lint.js:83:5
    at Array.forEach (native)
    at Object.<anonymous> (/Users/example/node_modules/sass-lint/bin/sass-lint.js:82:16)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

If you're using a IDE plugin have you tried the CLI too?

CLI Tool

**Comment by DanPurdy** [comment]
Hi @mattmischuk This is a parse error with the AST we use [gonzales-pe](https://github.com/tonyganch/gonzales-pe) Would you mind opening an issue over there and linking back to this issue.

**Comment by mattmischuk** [comment]
https://github.com/tonyganch/gonzales-pe/issues/253

**Comment by Kunepro** [comment]
Hello, I use Angular-CLI with sass and I've a very similar issue with /deep/, I guess that I ask to gonzales-pe as well?

**Comment by mattmischuk** [comment]
@DanPurdy https://github.com/tonyganch/gonzales-pe/issues/253 was resolved.

**Comment by DanPurdy** [comment]
Yep, @bgriffith has been plowing through the issues recently. Once a new Gonzales update is ready we'll close this ticket and update sass-lint

**Comment by mattmischuk** [comment]
Thanks, look forward to it.

**Comment by bgriffith** [comment]
This should now be fixed as of `v1.11.1` - if you updated recently, you may need to run `npm i` again
